glossary_globus.tex

```diff
@@ -0,0 +1,17 @@
+\newglossaryentry{Collection}
+{ name={Collection},
+  description={A collection is a named location containing data you can access with Globus. Collections can be hosted on many different kinds of systems, including campus storage, HPC clusters, laptops, Amazon S3 buckets, Google Drive, and scientific instruments. When you use Globus, you don’t need to know a physical location or details about storage. You only need a collection name. A collection allows authorized Globus users to browse and transfer files. Collections can also be used for sharing data with others, for data publication, and for enabling discovery by other Globus users.},
```

Comment:
make sure lines are no longer than 120 chars. it is very hard comment on very long lines, adn it does not matter for the formatting to split them over multiple lines